### PR TITLE
refactor(setting): add prefix "format_" for format related settings.

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

see https://github.com/datafuselabs/databend/discussions/7935#discussioncomment-3758425

Fixes #issue

### CODE REVIEW

File: docs/doc/21-load-data/00-local.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ CREATE TABLE books
 <TabItem value="csv" label="CSV">
 
 ```shell title='Request'
-curl -XPUT 'http://root:@127.0.0.1:8081/v1/streaming_load' -H 'insert_sql: insert into book_db.books format CSV' -H 'skip_header: 0' -H 'field_delimiter: ,' -H 'record_delimiter: \n' -F 'upload=@"./books.csv"'
+curl -XPUT 'http://root:@127.0.0.1:8081/v1/streaming_load' -H 'insert_sql: insert into book_db.books format CSV' -H 'format_skip_header: 0' -H 'format_field_delimiter: ,' -H 'format_record_delimiter: \n' -F 'upload=@"./books.csv"'
```
Comment (Xuanwo) [inline docs/doc/21-load-data/00-local.md:75]: So our SQL is not changed?

Comment (youngsofun) [reply docs/doc/21-load-data/00-local.md:75]: no,  copy do not need prefix `FILE_FORMAT = (type = 'CSV' field_delimiter = '\t')`

File: tests/suites/1_stateful/01_load_v2/01_0004_streaming_variant_load.sh [hunk 2]
```diff
@@ -12,32 +12,32 @@ aws --endpoint-url http://127.0.0.1:9900/ s3 cp s3://testbucket/admin/data/json_
...
-	echo "The downloaded dataset has been corrupted, please remove and fetch it again."
-	exit 1
-fi
-
-echo "ff2f75d3107759cc09d79687143831fce9f38455de001c3e170c83f449492ec6 /tmp/json_sample.ndjson" | sha256sum --check > /dev/null 2>&1
-if [ $? -ne 0 ]; then
-	echo "The downloaded dataset has been corrupted, please remove and fetch it again."
-	exit 1
-fi
+## do the Data integrity check
```
[the remark this answers is not in the document]
Comment (youngsofun) [reply tests/suites/1_stateful/01_load_v2/01_0004_streaming_variant_load.sh]: fixed, thank you

[the remark this answers is not in the document]
Comment (youngsofun) [reply tests/suites/1_stateful/01_load_v2/01_0004_streaming_variant_load.sh]: seems no sha256sum on mac (home brew)

**Review by Xuanwo — APPROVED** [review]

**Review by sundy-li — APPROVED** [review]

**Pull request merged**